coretex/networking/network_manager_base.py

```diff
@@ -407,7 +412,9 @@ def formData(
             headers = self._headers("multipart/form-data")
             del headers["Content-Type"]
 
-            return self.request(endpoint, RequestType.post, headers, body = params, files = filesData)
+            timeout: TimeoutType = (600, 300) if len(files) > 0 else REQUEST_TIMEOUT
```

Comment:
Why is the connection timeout for HTTP so long?

Reply:
Testing on 128MB file uploads, 60s was just above the thresholds where the request would succeed. I 10x-ed this value to account for slower user upload speed

Reply:
Can we do another approach for this, as this is not optimal?